src/test/java/com/xceptance/neodymium/tests/TestDataStatementTest.java

```diff
@@ -267,10 +268,9 @@ public void testMethodDefaultTwoDataSet() throws Throwable
     {
         // Two data sets and @Testdata on method
         String[] expected = new String[]
-        {
-          "test1 :: Data set 1 / 2",
-          "test1 :: Data set 2 / 2"
-        };
+            {
+              "test1 :: Data set 1 / 2", "test1 :: Data set 2 / 2"
```

Comment:
Was this change intended needed? Can we go back to the old format for better readability.

Reply:
Formatting changes reverted